data/core/init.lua

```diff
@@ -782,9 +415,10 @@ function core.init()
       update_recents_project("remove", project_dir)
     end
     project_dir_abs = system.absolute_path(".")
-    if not core.set_project_dir(project_dir_abs, function()
+    local status, err = pcall(core.set_project, project_dir_abs)
+    if status then
       got_project_error = not core.load_project_module()
-    end) then
+    else
       system.show_fatal_error("Lite XL internal error", "cannot set project directory to cwd")
```

Comment:
This message is not correct anymore.

Reply:
This message is not necessary at all any more; as there's no syscall that can fail here.